kazoo/recipe/lock.py

```diff
@@ -207,13 +200,12 @@ def _acquire_lock():
                 )
             except RetryFailedError:
                 pass
-            except KazooException:
+            except KazooException as ex:
```

Comment:
[nit] ditto about the consistency of using `err` vs `ex`.

Reply:
There's not a lot of consistency overall. I was inferring the name from the class name, `...Exception as ex`, `...Error as err`.

What do you think?

Reply:
I think that makes perfect sense 😄 

I did a quick poll of the codebase and it is not very consistent indeed.

We currently mostly have `err`, `exc` and (my "favorite") the ambivalent `e`.

   1 ex
   6 err
   6 exc
   7 e

I would suggest we then follow your model but with `*Error` / `*Exception` aligned with `err` / `exc` respectively.

We can remediate the others overtime.
Would you be OK with that?